docs/csharp/language-reference/keywords/fixed-statement.md

```diff
@@ -19,6 +19,12 @@ You can initialize a pointer by using an array, a string, a fixed-size buffer, o
 
 [!code-csharp[Initializing fixed size buffers](../../../../samples/snippets/csharp/keywords/FixedKeywordExamples.cs#2)]
 
+Starting with C# 7.3, the `fixed` statement operates on additional types beyond arrays, strings, fixed-size buffers, or unmanaged variables. Any type that implements a method named `DangerousGetPinnableReference` can be pinned. The `DangerousGetPinnableReference` must return a `ref` variable to an unmanaged type. See the topic on [pointer types](../../programming-guide/unsafe-code-pointers/pointer-types.md) for more information. The .NET Types <xref:System.Span%601?displayProperty=nameWithType>, <xref:System.ReadonlySpan%601?displayProperty=nameWithType> introduced in .NET Standard 2.0 make use of this pattern and can be pinned. This is shown in the following example:
+
+[!code-csharp[Accessing fixed memory](../../../../samples/snippets/csharp/keywords/FixedKeywordExamples.cs#FixedSpan)]
+
+If you are created types that should participate in this pattern, see <xref:System.Span%601.DangerousGetPinnableReference?displayProperty=nameWithType> for an example of implementing the pattern.
```

Comment:
DangerousGetPinnableReference isn't currently a public type in [Span\<T>](https://docs.microsoft.com/dotnet/api/system.span-1).

Reply:
It seemed to be public: [DangerousGetPinnableReference](https://docs.microsoft.com/en-us/dotnet/api/system.span-1.dangerousgetpinnablereference?view=netcore-2.0#System_Span_1_DangerousGetPinnableReference)